### PR TITLE
Tweak layout of template selector

### PR DESCRIPTION
**Pull request by yakshaG**
Tweak template selector layout
- slider arrow placement and list item layout
- selected tick placement
- selected template borders

### CODE REVIEW

**Comment by gopal1996** [comment]
template selector arrow space looks uneven @yakshaG

File: src/modules/builder/nav-bar/components/TemplatesSlider.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const TemplateSlider = () => {
       splideInstanceRef.current = new SplideCore(targetElement, {
         perPage: 2,
         pagination: false,
-        gap: '33px',
+        gap: '-72px',
```
Comment (gopal1996) [inline src/modules/builder/nav-bar/components/TemplatesSlider.tsx:24]: why negative?

File: src/modules/builder/nav-bar/components/TemplatesSlider.tsx [hunk 2]
```diff
@@ -47,11 +47,9 @@ export const TemplateSlider = () => {
             fill: '#000000',
           },
           '.splide__arrow--prev': {
-            left: '-2.2rem',
             backgroundColor: 'transparent',
           },
           '.splide__arrow--next': {
-            right: '-2.2rem',
```
Comment (gopal1996) [inline src/modules/builder/nav-bar/components/TemplatesSlider.tsx:54]: somewhat arrow space looks uneven @yakshaG

File: src/modules/builder/nav-bar/components/TemplatesSlider.tsx [hunk 3]
```diff
@@ -93,21 +91,21 @@ export const TemplateSlide = ({
...
-          isActive ? 'bg-resume-50 border-resume-500' : 'border-[#a9a9a9]'
+          isActive ? 'bg-resume-50 border-resume-800' : 'border-resume-200'
         }`}
       >
         <Image src={thumbnail} alt={name} layout="fill" />
 
         {isActive && (
-          <div className="absolute top-3 right-3">
-            <Image src={'/icons/selected-tick.svg'} alt="logo" width={'28px'} height={'20px'} />
+          <div className="absolute top-1 right-1 bg-white">
```
Comment (gopal1996) [inline src/modules/builder/nav-bar/components/TemplatesSlider.tsx:107]: while the image is loading in the slow network, white background is shown around tick icon

File: src/modules/builder/nav-bar/components/TemplatesSlider.tsx (hunk 1 above)
Comment (yakshaG) [reply src/modules/builder/nav-bar/components/TemplatesSlider.tsx:24]: I increased the container width, so the gap between items in the slider list looked huge with any positive values. Not sure how to adjust it, without using negative values.

File: src/modules/builder/nav-bar/components/TemplatesSlider.tsx (hunk 3 above)
Comment (yakshaG) [reply src/modules/builder/nav-bar/components/TemplatesSlider.tsx]: Removed redundant `resume-bg-50` for selected template and added `rounded-full` to the tick image div. Now the bg will be white but rounded and will not be visible as the PNG of the image also loads simultaneously.

File: src/modules/builder/nav-bar/components/TemplatesSlider.tsx (hunk 2 above)
Comment (yakshaG) [reply src/modules/builder/nav-bar/components/TemplatesSlider.tsx:54]: Yeah fixed

**Comment by yakshaG** [comment]
> template selector arrow space looks uneven @yakshaG

Yeah fixed.

File: src/modules/builder/nav-bar/components/TemplatesSlider.tsx [hunk 4]
```diff
@@ -93,21 +91,21 @@ export const TemplateSlide = ({
...
+          isActive ? 'border-resume-800' : 'border-resume-200'
         }`}
       >
         <Image src={thumbnail} alt={name} layout="fill" />
 
         {isActive && (
-          <div className="absolute top-3 right-3">
-            <Image src={'/icons/selected-tick.svg'} alt="logo" width={'28px'} height={'20px'} />
+          <div className="absolute top-1 right-1 bg-white rounded-full">
+            <Image src={'/icons/selected-tick.svg'} alt="logo" width={'24px'} height={'24px'} />
```
Comment (gopal1996) [inline src/modules/builder/nav-bar/components/TemplatesSlider.tsx:108]:
```suggestion
            <Image src={'/icons/selected-tick.svg'} className="rounded" alt="logo" width={'24px'} height={'24px'} />
```

File: src/modules/builder/nav-bar/components/TemplatesSlider.tsx [hunk 5]
```diff
@@ -93,21 +91,21 @@ export const TemplateSlide = ({
...
+          isActive ? 'border-resume-800' : 'border-resume-200'
         }`}
       >
         <Image src={thumbnail} alt={name} layout="fill" />
 
         {isActive && (
-          <div className="absolute top-3 right-3">
-            <Image src={'/icons/selected-tick.svg'} alt="logo" width={'28px'} height={'20px'} />
+          <div className="absolute top-1 right-1 bg-white rounded-full">
+            <Image src={'/icons/selected-tick.svg'} className="rounded" alt="logo" width={'24px'} height={'24px'} />
```
Comment (gopal1996) [inline src/modules/builder/nav-bar/components/TemplatesSlider.tsx:108]:
```suggestion
            <Image src={'/icons/selected-tick.svg'}  alt="logo" width={'24px'} height={'24px'} />
```

File: src/modules/builder/nav-bar/components/TemplatesSlider.tsx (hunk 1 above)
Comment (siva-kannan3) [reply src/modules/builder/nav-bar/components/TemplatesSlider.tsx]: made changes in the gap part

File: src/modules/builder/nav-bar/components/TemplatesSlider.tsx [hunk 6]
```diff
@@ -92,22 +90,20 @@ export const TemplateSlide = ({
...
+          onChangeTemplate(id);
+        }}
       >
         <Image src={thumbnail} alt={name} layout="fill" />
 
         {isActive && (
-          <div className="absolute top-3 right-3">
-            <Image src={'/icons/selected-tick.svg'} alt="logo" width={'28px'} height={'20px'} />
+          <div className="absolute top-1 right-1 bg-white rounded-full">
+            <Image src={'/icons/selected-tick.svg'}  alt="logo" width={'24px'} height={'24px'} />
```
Comment (gopal1996) [inline src/modules/builder/nav-bar/components/TemplatesSlider.tsx:106]:
```suggestion
            <Image src={'/icons/selected-tick.svg'} alt="logo" width={'24px'} height={'24px'} />
```

**Pull request merged**